flang/lib/Lower/OpenMP.cpp

```diff
@@ -110,6 +110,34 @@ static void gatherFuncAndVarSyms(
   }
 }
 
+static Fortran::lower::pft::Evaluation *
+getCollapsedEval(Fortran::lower::pft::Evaluation &eval, int collapseValue) {
```

Comment:
Nit: Maybe `getCollapsedLoopEval` would be slightly more self-explanatory?

Reply:
I'll make this change in another commit.